app/src/main/res/values/dimens.xml

```diff
@@ -5,6 +5,7 @@
   <dimen name="fab_margin">16dp</dimen>
   <dimen name="headingText">16sp</dimen>
 
+  <dimen name="cameraIcon">30dp</dimen>
```

Comment:
@pa1pal Ok. i will set them pro-grammatically and send a PR for this. Thanks for Review